_posts/2020-04-26-recent-articles.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Recent articles about Signal K
+layout: post
+author: Paul Sumpner
+email: paul at digitalyacht dot co dot uk,
+gravatar: https://avatars1.githubusercontent.com/u/10165637?s=88&u=c6ea699fd10c7566bbb6f4c4c18a0abeb7d344cd&v=4
+status: published
+---
+
+The message is gradually getting out there about the benefits of Signal K and there have been some really interesting articles published recently on other blogs, and we thought we would give you links to them here.
+
+### Victron Blog - Sailing with Signal K
+
+Victron the innovative and highly respected Dutch manufacturer of energy products has embraced Signal K and this post tells the story of sailing boat Taku Moe and how the owners Stefanie and Heike have installed and used Signal K with their Victron system.
+
+[https://www.victronenergy.com/blog/2019/06/13/sailing-with-signal-k/](https://www.victronenergy.com/blog/2019/06/13/sailing-with-signal-k/)
```

Comment:
While a cool entry this is now close to a year old and to me hardly counts as _recent_. Something else instead?

Reply:
Panbo article?
Or 
https://bareboat-necessities.github.io/my-bareboat/

Reply:
In an ideal world the Victron article would have been posted in our blog at the same time Victron published it but it wasn’t. I felt it was important enough to still have it in our blog even if it is not bang up to date.

I could swap it for the bare necessities post in this “recent” blog post and create a back dated post on the Victron article, I think it justifies this and should be recorded on our site.

Reply:
i don’t think our blog is such a reference that backdating is needed, swap it for barenecessities and let’s move forward?